resilience4j-core/src/main/java/io/github/resilience4j/core/IntervalFunction.java

```diff
@@ -1,11 +1,11 @@
-package io.github.resilience4j.retry;
+package io.github.resilience4j.core;
```

Comment:
does this code compile ? the package is different from the folder structure.

Reply:
Yes it does, you can try with ./gradlew build 
If you look at the full filename in github (hover over the header here) you can see I moved this file